packages/pigeon/example/app/ios/Runner/EventChannelMessages.g.swift

```diff
@@ -23,6 +23,67 @@ private func nilOrValue<T>(_ value: Any?) -> T? {
   return value as! T?
 }
 
+func deepEqualsEventChannelMessages(_ lhs: Any?, _ rhs: Any?) -> Bool {
+  let cleanLhs = nilOrValue(lhs) as Any?
+  let cleanRhs = nilOrValue(rhs) as Any?
+  switch (cleanLhs, cleanRhs) {
+  case (nil, nil):
+    return true
+
+  case (nil, _), (_, nil):
+    return false
+
+  case is (Void, Void):
+    return true
+
+  case let (cleanLhsHashable, cleanRhsHashable) as (AnyHashable, AnyHashable):
+    return cleanLhsHashable == cleanRhsHashable
```

Comment:
Hashable doesnt mean equal

Reply:
hashable inherits from equitable, there is no AnyEquitable.

Reply:
sorry i misread. i thought you were comparing 2 hash values.